### ISSUE TITLE
recent searches prompting

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#8487]
Search for something.
Then later want to search again for it.
But there is no drop down search history etc.

**Comment by matkoniecz** [comment]
are you talking about preset search history?

**Comment by matkoniecz** [comment]
if it is for presets then such dropdown would obscure first found preset entry

**Comment by tyrasd** [comment]
Yeah, at first glance I also don't see this coming in useful too often while mapping. If there was a common use case where it makes sense to have such a feature, we can reconsider it.

**Comment by jidanni** [comment]
@m... [you are not giving me an opportunity to respond](https://ux.stackexchange.com/questions/155316/how-to-improve-flawed-blocking-algorithm).

I was thinking something like

<img width="539" height="518" alt="Image" src="https://github.com/user-attachments/assets/37b6c873-3ffe-4342-aac7-0d5a4bfb12ef" />
